.github/workflows/make_release.yml

```diff
@@ -0,0 +1,61 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Create Release
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    if: github.repository == 'napari/napari'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Install Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Build Distribution
+        run: |
+          pip install wheel
+          python setup.py sdist bdist_wheel
+      - name: Find Release Notes
+        run: |
+          TAG="${GITHUB_REF/refs\/tags\/v/}"  # clean tag
+          VER="${TAG/rc*/}"  # remove pre-release identifier
+          RELEASE_FILEPATH=docs/release/release_${VER//./_}.rst
+          # https://help.github.com/en/actions/reference/workflow-commands-for-github-actions
+          echo "::set-env name=tag::$TAG"
+          echo "::set-env name=release_filepath::$RELEASE_FILEPATH"
+      - name: Create Release
+        id: create_release
+        uses: jbolda/create-release@v1.1.0  # switch to actions/create-release@latest once PR merged
```

Comment:
I'm assuming you mean PR https://github.com/actions/create-release/pull/50 ? can you add that url for easy lookup?

Reply:
I included a shorthand instead of the full url, hope that's okay :)